### PR TITLE
Sample of creating factory pool using web3js

### PR DESCRIPTION
**Pull request by grandizzy**
- in a terminal run: brownie console
run('sdk_setup')

- in a different window (do not close brownie console), cd to script dir and npm install
node createERC20Pool.js

This will create a COMP/DAI ERC20 pool using properties within .env file generated in scripts dir. The private key from env file can be imported in Metamask for further tests

### CODE REVIEW

**Review by MikeHathaway — APPROVED** [review]
LGTM - should this be in the sdk repository instead?

**Comment by grandizzy** [comment]
> LGTM - should this be in the sdk repository instead?

Hm, yeah, that would be a better place. Just wanted to have a quick start with pool creation, let's discuss how to accommodate in sdk repo (probably the brownie script could remain here but spit the .env file and move abi in an external location from where sdk could pick it up?)

File: scripts/createERC20Pool.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  );
+  web3.eth.accounts.wallet.add(signer);
+  // Creating a Contract instance
+  const contract = new web3.eth.Contract(
+    abi,
+    // Replace this with the address of your deployed contract
+    process.env.ERC20_FACTORY
+  );
+  const interestRate = web3.utils.toWei(String(0.05), 'ether');
+  const tx = contract.methods.deployPool(process.env.COMP, process.env.DAI, interestRate);
```
Comment (xdemocle) [inline scripts/createERC20Pool.js:27]: this is the key to success ahahah @grandizzy

**Review by xdemocle — APPROVED** [review]

**Comment by xdemocle** [comment]
you can merge it if you want. @grandizzy I understood now. thanks a lot

**Pull request merged**